src/code/z_collision_check.c

```diff
@@ -1,4 +1,5 @@
 #include "global.h"
+#include "prevent_bss_reordering.h"
```

Comment:
I just wanted to say the order in which you include this header affects bss reorder too. If you include it like this you may get different results:
```c
#include "prevent_bss_reordering.h"
#include "global.h"
```
I just wanted you to know this, you may need it in the future.

Reply:
Yes, the file itself says that it should go before any global header includes.

Reply:
That's really useful to know. Too useful actually, because moving it before `"global.h"` as suggested causes a NON_MATCH. However, moving it to the next file in the `spec` list, `code_800E8EA0` before `"global.h"` is alternative that matches OK and keeps it above `"global.h"`. Which would be preferred?